src/no/runsafe/entitycontrol/pets/CompanionPetAnimal.java

```diff
@@ -30,8 +32,9 @@ public CompanionPetAnimal(World world)
 			e.printStackTrace();
 		}
 
+		this.world = world;
 		goalSelector.a(0, new PathfinderGoalFloat(this));
-		setAge(-1000);
+		setAge(-2000000000);
```

Comment:
Isn't there like an Int.MinValue you could use rather than an arbitrary negative value?